### ISSUE TITLE
unable to use wildcards with event store

### ISSUE THREAD

**Issue by stephen-dahl** [kubemq-io/kubemqctl#7]
I have a cluster running on a clean docker desktop k8s instance. whenever I use a wildcard with event store the connection closes.
```
kubemqctl create cluster
kubemqctl es a *
Connecting To Kubernetes Cluster... Current Kubernetes cluster context connection: docker-desktop
-> connected to kubemq/kubemq-cluster-1 at gRPC Port 53169:50000 Rest Port 53170:9090 Api Port 53171:8080, ok
Adding '*' to attach list
[events_store]  [*]  kubemq-cluster-1 Server connected.
Error: attach web socket reading, websocket: close 1006 (abnormal closure): unexpected EOF
```
`kubemqctl es a test` works as expected

**Comment by kubemq** [comment]
kubemq in event store is not supporting wildcards as designed

**Comment by stephen-dahl** [comment]
looks like the examples need updated, it is showing several event commands under the examples for events_store

```
kubemqctl es a -h
Attach command allows to display 'events store' channel content for debugging proposes
Usage:
  kubemqctl events_store attach [flags]
Aliases:
  attach, a, att, at
Examples:
Examples:
        # Attach to all events store channels and output running messages
        kubemqctl events attach *
        # Attach to some-events-store 'events store' channel and output running messages
        kubemqctl events_store attach some-events-store
        # Attach to some-events-store1 and some-events-store2 'events store' channels and output running messages
        kubemqctl events attach some-events-store1 some-events-store2
        # Attach to some-events-store 'events store' channel and output running messages filter by include regex (some*)
        kubemqctl events attach some-events -i some*
        # Attach to some-events-store 'events store' channel and output running messages filter by exclude regex (not-some*)
        kubemqctl events attach some-events -e not-some*
```

**Comment by kubemq** [comment]
fixed on v2.3.1
